docs/content/tutorials/composite-recipe/index.md

```diff
@@ -0,0 +1,180 @@
+Since the `bicepconfig.json` file was modified in the previous tutorial, it does not need to be further modified.
+
+## Step 3: Create, publish, and register the composite Recipe
+
+The previous tutorial demonstrated deploying a resource using Terraform or Bicep resource providers such as the Kubernetes provider. This tutorial creates a composite Recipes which uses only built-in Radius types. These composite Recipes are only possible using Bicep.
+
+1. Create a new file called `webservices.bicep` and add the following:
+
+   {{% rad file="snippets/webservices.bicep" lang="bicep" embed=true %}}
+
+   Notice that the `container` and `connections` properties are passed to the `Applications.Core/containers` built-in type using a single line.
+
+1. Publish the Recipe to an OCI registry. Make sure to replace `host` and `registry` with your container registry.
+
+   ```bash
+   rad bicep publish --file webservices.bicep --target br:<host>/<registry>/webservices:latest
+   ```
+
+1. Register the Bicep template as the `default` Recipe in the `default` environment.
+
+   ```bash
+   rad recipe register default --environment default \
+     --resource-type Radius.Resources/webServices \
+     --template-kind bicep \
+     --template-path <host>/<registry>/webservices:latest
+   ```
+
+   You can confirm the Recipe was registered using `rad recipe list` command.
+
+## Step 4: Replace the container resource with a web service
+
+1. Using the same `app.bicep` file, change the resource type for the `demo` resource from `Applications.Core/containers` to `Radius.Resources/webServices`.
+
+   ```diff
+   - resource demo 'Applications.Core/containers@2023-10-01-preview' = {
+   + resource demo 'Radius.Resources/webServices@2023-10-01-preview' = {
+   ```
+
+1. Add `ingress: true` to the demo resource so that a Gateway gets deployed as part of the web service. The web service also needs an environment property similar to the database custom resource.
+
+   ```diff
+   resource demo 'Radius.Resources/webServices@2023-10-01-preview' = {
+     name: 'demo'
+       properties: {
+         application: application
+   +     environment: environment
+   +     ingress: true
+   ```
+
+1. Since the web service resource type has a build-in Gateway, remove the Gateway resource.
+
+   ```diff
+   - resource gateway 'Applications.Core/gateways@2023-10-01-preview' = {
+   -   name: 'gateway'
+   -   properties: {
+   -     application: application
+   -     routes: [
+   -       {
+   -         path: '/'
+   -         destination: 'http://demo:3000'
+   -       }
+   -     ]
+   -   }
+   - }
+   ```
+
+1. Run the application using `rad run`. 
+
+   ```bash
+   rad deploy app.bicep
+   ```
+
+1. Unlike before, the Gateway URL is not automatically shown. Use the `rad resource show` command to get the URL of the application. 
+
+   ```bash
+   rad resource show Applications.Core/gateways gateway -o json | grep url
+   ```
+
+1. Open the URL in your browser.
+
+## Step 6: Clean up
```

Comment:
I think this set of clean up steps is a bit extreme - I recommend stopping short of deleting the default env, resource groups, and uninstalling Radius so that they still have those in place in case they want to try other tutorials or guides in the docs.